ComposeSample/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Please, @joyl1216 remove the `.idea` files from the repo so we don't track them here (there are specific to every user IDE configuration).